source/extensions/filters/listener/http_inspector/http_inspector.cc

```diff
@@ -19,8 +19,7 @@ namespace Extensions {
 namespace ListenerFilters {
 namespace HttpInspector {
 
-Config::Config(Stats::Scope& scope)
-    : stats_{ALL_HTTP_INSPECTOR_STATS(POOL_COUNTER_PREFIX(scope, "http_inspector."))} {}
```

Comment:
does this risk breaking existing integrations that expect http_inspector. stats without the ip address?